Add integration test for virt_pool module

saito-hideki opened the pull request:
##### SUMMARY
Add integration test for virt_pool module

- Fixes #7 

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
- community.libvirt.virt_pool

##### ADDITIONAL INFORMATION
- This test is using a directory based storage pool

Andersson007 reviewed (commented):
@saito-hideki thank you for adding the integration tests! This is, in fact, very valuable!

if the covered things support `check_mode`, I would suggest adding tasks running with `check_mode: yes` and corresponding idempotency checks.

@saito-hideki that do you think?

saito-hideki commented:
@Andersson007 thank you for your advice :)
Yeah, that's true. I'll append some tests with `check_mode: true`.

saito-hideki commented:
@Andersson007 I have added `check_mode` tasks to the integration test. Also, I have removed Python2 environment from CI docker tests on devel branch.
I would appreciate it if you could review this PR again :)
